robottelo/constants/__init__.py

```diff
@@ -1761,6 +1761,12 @@
     "username": "random_name",
 }
 
+UPDATE_USER = {'realm': "realm_name", "userId": "user_id"}
```

Comment:
Does that need to be in constants?
Seems you only use dict keys, as you replace values in fixture.

Either way, this name is too generic and it's hard to make connection between name and actual content. Perhaps something like `RHSSO_USER_KEYS_TO_UPDATE` would be more fitting.

Reply:
Just to make easier I making it as `RHSSO_USER_UPDATE` not too long and not generic as well